inc/issue.class.php

```diff
@@ -182,18 +182,15 @@ public static function getSyncIssuesRequest() : AbstractQuery {
             ],
             [
                'TABLE' => new QuerySubQuery([
-                  'SELECT' => '*',
-                  'FROM' => new QuerySubQuery([
-                     'SELECT' => ['users_id', $ticketFk],
-                     'DISTINCT' => true,
-                     'FROM'  => $ticketUserTable,
-                     'WHERE' => [
-                        'type' => CommonITILActor::REQUESTER,
-                     ],
-                     'ORDER' => ['id ASC'],
-                  ], 'inner_glpi_tickets_users'),
-                  'GROUPBY' => 'tickets_id'
-               ], 'glpi_tickets_users'),
+                  'SELECT' => ['users_id', $ticketFk],
+                  'DISTINCT' => true,
+                  'FROM'  => $ticketUserTable,
+                  'WHERE' => [
+                     'type' => CommonITILActor::REQUESTER,
+                  ],
+                  'GROUPBY' => 'tickets_id',
+                  'ORDER' => ['id ASC']
+               ], $ticketUserTable),
```

Comment:
while searching to make the query full_group_by compliant, I found it is possible to remove 1 nesting of SELECT.